blog/_posts/2023-04-06-march-2023-scala-tooling-summit.md

```diff
@@ -0,0 +1,357 @@
+---
+layout: blog-detail
+post-type: blog
+by: Chris Kipp and the Scala Center team
+title: March 2023 - Scala Tooling Summit
+description: A recap of the recent Scala Tooling Summit in Lausanne, Switzerland.
+---
+
+<img style="max-height: 350px; display: block; margin: auto;" src="{{ site.baseurl }}/resources/img/march-2023-tooling-summit-stairs.jpg" alt="Scala stairs with people standing on the stairs">
```

Comment:
I don't love this, but the styling for images in blogs is massive. I saw in others that we seem to rely on inline styling, so I just followed suit here. However, we should probably change this up in the future and actually correctly style images with css.

This ends up looking like this:
<img width="1204" alt="Screenshot 2023-04-06 at 16 08 44" src="https://user-images.githubusercontent.com/13974112/230402854-95aed441-daa8-4ed1-b50b-e74b5dea6584.png">